packages/x-date-pickers/src/MobileTimePicker/MobileTimePicker.tsx

```diff
@@ -47,10 +47,6 @@ const MobileTimePicker = React.forwardRef(function MobileTimePicker<TDate>(
         ...resolveComponentProps(defaultizedProps.slotProps?.field, ownerState),
         ...extractValidationProps(defaultizedProps),
         ref,
```

Comment:
I would like to also handle the ref in `useDesktopPicker` / `useMobilePicker` / ...
But it creates a TS error when I stop passing it through the `slotProps` in the picker file.
And I struggle to fix it, so I did not touch this behavior for now.